lib/workload/stateful/filemanager/deploy/stack/filemanager_stack.ts

```diff
@@ -4,18 +4,21 @@ import * as iam from 'aws-cdk-lib/aws-iam';
 import { RustFunction, Settings as CargoSettings } from 'rust.aws-cdk-lambda';
 import { Architecture } from 'aws-cdk-lib/aws-lambda';
 import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as rds from 'aws-cdk-lib/aws-rds';
 import { CfnBucket } from 'aws-cdk-lib/aws-s3';
 import * as sqs from 'aws-cdk-lib/aws-sqs';
 import * as lambdaDestinations from 'aws-cdk-lib/aws-lambda-destinations';
 import * as lambdaEventSources from 'aws-cdk-lib/aws-lambda-event-sources';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
 
 /**
  * Common settings for the filemanager stack.
  */
 interface Settings {
   database_url: string;
-  endpoint_url: string;
-  force_path_style: boolean;
+  endpoint_url?: string;
+  force_path_style: boolean; //FIXME: Does this refer to buckets? We cannot do this forever as AWS will move to domain style?
```

Comment:
I think you can remove any `path_style` settings because it was only was only used for localstack.